R/deployApp.R

```diff
@@ -437,6 +440,30 @@ deployApp <- function(appDir = getwd(),
   invisible(deploymentSucceeded)
 }
 
+
+getPython <- function(path) {
```

Comment:
Idiomatically `path` would default to NULL here so that you can invoke a bare `getPython()` to load the default from the environment.